rules.ini

```diff
@@ -44,7 +44,16 @@ ChromeEngine[] = \.rpack$
 Clausewitz[] = (?:^|/)common/defines\.(?:lua|txt)$
 Clausewitz[] = ^(?:game/)?common/defines/00_defines\.txt$
 ClickTeamFusion = (?:^|/)mmfs2\.dll$
-Cocos2d = (?:^|/)libcocos2d\.dll$
+Cocos[] = (?:^|/)libcocos2d\.dll$
+Cocos[] = (?:^|/)Cocos2d\.lua$
+Cocos[] = (?:^|/)jsb-adapter/jsb-engine\.jsc?$
```

Comment:
it only matched 
* `jsb-engine.js` - 78 matches
```suggestion
Cocos[] = (?:^|/)jsb-adapter/jsb-engine\.js$
```

Reply:
`.jsc` is needed when developer enables `Encrypt JS` in Cocos Creator build panel. 
<img width="767" alt="WeChat88c7c695a491b0cf80172052708e560b" src="https://user-images.githubusercontent.com/493372/196868599-afb2c805-3643-4dbb-82f5-1314526f1546.png">

If it's enabled, all js files will be encrypted and use `.jsc` as its file suffix.